packages/x-components/src/components/base-grid.vue

```diff
@@ -34,6 +34,7 @@
   import { Component, Prop } from 'vue-property-decorator';
   import { toKebabCase } from '../utils/string';
   import { GridItem, VueCSSClasses } from '../utils/types';
+  import {XInject} from "./decorators/injection.decorators";
```

Comment:
format the code using prettier, because we have just moved this project to the mono-repo, the format/lint on commit is now broken, so until we fix it we have to do it manually